templates/ui-frameworks/vue/web-app/ui/src/ClientProvider.vue.hbs

```diff
@@ -0,0 +1,25 @@
+<template>
+  <progress v-if="loading"></progress>
+  <div v-else-if="error" class="alert">
+    Error connecting to Holochain: {{{{raw}}}} {{error.message}}.{{{{/raw}}}}
+  </div>
+  <slot v-else-if="client"></slot>
+
+  <button v-if="client && IS_HOLO" @click="signOut">Sign Out</button>
+</template>
+
+<script setup lang="ts">
+{{#if holo_enabled}}
+import WebSdk from "@holo-host/web-sdk"
+{{/if}}
+import { useClientProvider{{#if holo_enabled}}, IS_HOLO{{/if}} } from './clientProvider';
```

Comment:
We're always using `IS_HOLO` in the template, so either that should be conditional, or this import should not be.

Reply:
The `IS_HOLO` const will only be available if the happ was scaffolded with the `--holo` flag

Reply:
oops was not looking at the recent commits. I see you put the sign out button around a conditional, so all good.